onnxruntime/test/providers/openvino/openvino_ep_ext_init.cc

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <filesystem>
+#include <map>
+#include <string>
+
+#include "core/session/onnxruntime_cxx_api.h"
+
+#include "test/util/include/test/test_environment.h"
+#include "test/unittest_util/qdq_test_utils.h"
+
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "onnxruntime_session_options_config_keys.h"
+
+using namespace ONNX_NAMESPACE;
+using namespace onnxruntime::logging;
+
+extern std::unique_ptr<Ort::Env> ort_env;
+
+class OVEP_ExtInit_Tests : public ::testing::TestWithParam<std::string> {};
+
+namespace {
+
+std::vector<uint8_t> LoadFileToMemory(const std::string& path) {
```

Comment:
How about returning std::optinal<std::vector<uint8_t>> so you can return {} if there is nothing?